packages/vaadin-lumo-styles/src/components/checkbox.css

```diff
@@ -242,7 +242,7 @@
 }
 
 :host([invalid]) [part='checkbox'] {
-  background: var(--_invalid-background);
+  background-color: var(--_invalid-background);
```

Comment:
The new linter rule highlighted this place, and manual testing confirmed that it does indeed cause an issue with the check element's color:

| background | background-color |
|-------------|-------------------|
| ![localhost_8000_dev_checkbox html](https://github.com/user-attachments/assets/effab4b4-133d-4578-954a-b11ea36fc490) | ![localhost_8000_dev_checkbox html (1)](https://github.com/user-attachments/assets/65fa53ff-3a39-44a9-9571-498c8c2a05c2) |

It's also unfortunate that this wasn't caught by the visual tests.